util/tls/tls_socket.cc

```diff
@@ -203,18 +203,44 @@ io::Result<size_t> TlsSocket::Recv(const io::MutableBytes& mb, int flags) {
 }
 
 io::Result<size_t> TlsSocket::WriteSome(const iovec* ptr, uint32_t len) {
+  // Chosen to be sufficiently smaller than the usual MTU (1500) and a multiple of 64.
+  constexpr size_t kBufferSize = 1408;
```

Comment:
are you sure that the encrypted stream + tls metadata does not go beyong 1500?

Reply:
Good question :) Checked again and it can go over 1500 if the TCP options field is really large. 
IP headers - 24 bytes
TCP headers - max 60 (typically around 20-30)
TLS headers - 21 bytes
I missed the IP headers size in my previous calculation, so I'm adjusting to 1392 now